stix_shifter/stix_transmission/src/modules/security_advisor/security_advisor_connector.py

```diff
@@ -0,0 +1,30 @@
+from ..base.base_connector import BaseConnector
+from .security_advisor_ping import SecurityAdvisorPing
+from .security_advisor_query_connector import SecurityAdvisorQueryConnector
+from .security_advisor_status_connector import SecurityAdvisorStatusConnector
+from .security_advisor_results_connector import SecurityAdvisorResultsConnector
+from .security_advisor_delete_connector import SecurityAdvisorDeleteConnector
+from .security_advisor_auth import SecurityAdvisorAuth
+
+class Connector(BaseConnector):
+    def __init__(self, connection, configuration ):
+        
+        auth  = {}
+        accountID = configuration.get("ibmCloudAccountID")
+        apiKey = configuration.get("ibmCloudApiKey") 
+        host = configuration.get("saAPIEndpoint")
+
+        if( not accountID or not apiKey or not host ):
```

Comment:
If you do this check at the initialization make sure you gracefully return the exception in the return object. simply raising exception just breaks out of the call. i would suggest you do this check in `SecurityAdvisorAuth`